qt/scientific_interfaces/Inelastic/Analysis/IndirectFitDataView.cpp

```diff
@@ -88,8 +111,14 @@ void IndirectFitDataView::setHorizontalHeaders(const QStringList &headers) {
 
   auto header = m_uiForm->tbFitData->horizontalHeader();
   header->setSectionResizeMode(0, QHeaderView::Stretch);
+
+  m_uiForm->tbFitData->setItemDelegateForColumn(headers.indexOf("StartX"),
+                                                std::make_unique<NumericInputDelegate>().release());
+  m_uiForm->tbFitData->setItemDelegateForColumn(headers.indexOf("EndX"),
+                                                std::make_unique<NumericInputDelegate>().release());
```

Comment:
I think we can use the `startXColumn()` and `endXColumn()` methods to get the index of the "StartX" and "EndX" columns